### ISSUE TITLE
Sourcify partially verification via web: 500 response on partially verified contract

### ISSUE THREAD

**Issue by nikitosing** [blockscout/blockscout#4428]
```2021-07-23T17:35:49.781 [error] #PID<0.11606.0> running BlockScoutWeb.Endpoint (connection #PID<0.10459.0>, stream id 25) terminated
Server: localhost:4000 (http)
Request: POST /verify_smart_contract/contract_verifications
** (exit) an exception was raised:
    ** (CaseClauseError) no case clause matching: {:error, "partial"}
        (block_scout_web 0.0.1) lib/block_scout_web/controllers/address_contract_verification_controller.ex:107: BlockScoutWeb.AddressContractVerificationController.verify_and_publish/3
        (block_scout_web 0.0.1) lib/block_scout_web/controllers/address_contract_verification_controller.ex:83: BlockScoutWeb.AddressContractVerificationController.create/2
        (block_scout_web 0.0.1) lib/block_scout_web/controllers/address_contract_verification_controller.ex:1: BlockScoutWeb.AddressContractVerificationController.action/2
        (block_scout_web 0.0.1) lib/block_scout_web/controllers/address_contract_verification_controller.ex:1: BlockScoutWeb.AddressContractVerificationController.phoenix_controller_pipeline/2
        (phoenix 1.5.6) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2
        (block_scout_web 0.0.1) lib/block_scout_web/endpoint.ex:1: BlockScoutWeb.Endpoint.plug_builder_call/2
        (block_scout_web 0.0.1) lib/plug/debugger.ex:136: BlockScoutWeb.Endpoint."call (overridable 3)"/2
        (block_scout_web 0.0.1) lib/block_scout_web/endpoint.ex:1: BlockScoutWeb.Endpoint."call (overridable 4)"/2
        (block_scout_web 0.0.1) lib/spandex_phoenix.ex:156: BlockScoutWeb.Endpoint.call/2
        (phoenix 1.5.6) lib/phoenix/endpoint/cowboy2_handler.ex:65: Phoenix.Endpoint.Cowboy2Handler.init/4
        (cowboy 2.8.0) e:/blocksckout/blockscout/deps/cowboy/src/cowboy_handler.erl:37: :cowboy_handler.execute/2
        (cowboy 2.8.0) e:/blocksckout/blockscout/deps/cowboy/src/cowboy_stream_h.erl:300: :cowboy_stream_h.execute/3
        (cowboy 2.8.0) e:/blocksckout/blockscout/deps/cowboy/src/cowboy_stream_h.erl:291: :cowboy_stream_h.request_process/3```

**Comment by nikitosing** [comment]
I suppose this issue was resolved by #4331

**Comment by Gam3356** [comment]
Thanks you guys! IPhone 12 Pro Max Ralo21LLC
